src/components/Topics.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { fetchTopics } from "../api";
+
+export default function Topics(){
+    const[topics,setTopics]=useState([])
+    useEffect(()=>{
+        fetchTopics().then(body=>{
+            setTopics(body.topics)
+        })
+    },[])
+
+    return (<div>
+        {topics.map(topic=>{
+         return <div key ={topic.slug}>
+         <Link to={"/articles/"+topic.slug} state={{topic:topic.slug}}> <h1>
+            {topic.slug}</h1>
```

Comment:
semantically you should only have one `<h1>` tag in an HTML document

You should use a different tag here!